libs/libsodium/Makefile

```diff
@@ -52,6 +52,7 @@ endef
 
 CONFIGURE_ARGS+= \
 	--disable-ssp \
+	--without-pthreads \
```

Comment:
You can select target branch when creating pull requests, and check the diff shown below before sending it.